src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/TaxonTranslation.xml

```diff
@@ -7,18 +7,7 @@
     <resource class="%sylius.model.taxon_translation.class%" shortName="TaxonTranslation">
         <attribute name="validation_groups">sylius</attribute>
 
-        <collectionOperations>
-            <collectionOperation name="admin_get">
-                <attribute name="method">GET</attribute>
-                <attribute name="path">/admin/taxon-translations</attribute>
-                <attribute name="normalization_context">
-                    <attribute name="groups">
-                        <attribute>admin:taxon:read</attribute>
-                        <attribute>admin:taxon_translation:read</attribute>
-                    </attribute>
-                </attribute>
-            </collectionOperation>
-        </collectionOperations>
+        <collectionOperations />
```

Comment:
What about normalization groups in this case?

Reply:
The endpoint itself went away. This was the only place where translations were treated differently and had an index, despite being included in the Taxon serialization.